tools/train.py

```diff
@@ -33,6 +33,9 @@
 from easycv.utils.config_tools import (CONFIG_TEMPLATE_ZOO,
                                        mmcv_config_fromfile, rebuild_config)
 
+# refer to: https://github.com/open-mmlab/mmdetection/pull/6867
```

Comment:
refer to can be added to pull request description rather than using code comment